lib/snapshot.js

```diff
@@ -12,7 +15,7 @@ class Snapshot {
     this.indexes = new Map()
     // name them .test.cjs so that nyc ignores them
     // base on main test filename, with sanitized argv
-    const args = process.argv.slice(2)
+    const args = argv.slice(2)
```

Comment:
Potentially a breaking change?  Someone could change process.argv before we get here and that currently would impact this function.

Reply:
The only way that could happen is if they're calling `new Test()` directly.  Since the `TAP` object is instantiated immediately, and all other tests are children of it, just calling `const t = require('libtap')` will result in the argv at that time being used.

But yes, if you are creating Test objects directly, this is a breaking change.  Will revert this bit.